### DISCUSSION TITLE
Bar chart does not show correct total number of screened records

### DISCUSSION THREAD

**Discussion by RiannevanderFeen** [asreview/asreview#1902] [Q&A] [answered]
Hello,

I had hoped to use the figures from the analytics in my paper, but I noticed that it does not show the correct total amount of screened records. 

I screened >10% in phase 2 (2738), but the figure (the bar chart!) states 2462 as you can see below:

![image](https://github.com/user-attachments/assets/25c1679d-9224-4212-94fb-57ae5589efc2)

**Comment by Rensvandeschoot** [comment]
Can you show us the descriptive statistics for your project?

**Comment by RiannevanderFeen** [comment]
Is this what you mean?

<img width="715" alt="image" src="https://github.com/user-attachments/assets/b8c92de3-fdcd-4f86-b3e0-120364ff8d0f">

**Reply by Rensvandeschoot** [reply]
thanks! and do you know how many records you selected as prior knowledge?

**Reply by RiannevanderFeen** [reply]
Yes, that's in fact 276 which adds up to 2738. But the issue is that I do not want to have to explain that in a paper when using the bar chart. 

At the same time, does this mean I have to screen 276 more records to come to the Phase 2 SAFE procedure stopping rule? 

In that case, it would be better to not show the 276 in the 'labeled records' tile. It's a bit confusing.

**Reply by Rensvandeschoot** [reply]
yes! see issue https://github.com/asreview/asreview/issues/1747 and the solution in https://github.com/asreview/asreview/pull/1766

**Comment by RiannevanderFeen** [comment]
It says that there should be a 'hide prior knowledge' switch and that this is 'on' by default, but I am not seeing a switch and looked in settings, details and analytics.

Where can I find the switch?

**Reply by Rensvandeschoot** [reply, marked as answer]
It will be available in v2...
